marketing_crm_partner/models/crm_lead.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2016 Tecnativa S.L. - Jairo Llopis
+# Copyright 2016 Tecnativa S.L. - Vicent Cubells
+# Copyright 2016 Tecnativa S.L. - David Vidal
+# Copyright 2018 Tecnativa S.L. - Cristina Martin R.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, models
+
+
+class CRMLead(models.Model):
+    _inherit = "crm.lead"
+
+    @api.model
+    def _prepare_customer_values(self, name, is_company, parent_id=False):
```

Comment:
You're changing the signature here, as the main method has this argument as optional:
```suggestion
    def _prepare_customer_values(self, name, is_company=False, parent_id=False):
```

Reply:
Thanks for your attention, I've fixed it up.